### PR TITLE
feat: Inform user when offline

### PR DESCRIPTION
**Pull request by dansvend**
Inform the user with no active network connection on the homepage and test screen before a test is fetched. Will fetch user info or test when reconnected.

Closes #121

### CODE REVIEW

**Comment by nesadrian** [comment]
@dansvend There is a couple of things I noticed when testing that you might have a look at 
- When I reconnect to wifi the no network banner is still showing.
- When offline I can still go to the submit feedback page but it will load indefinitely after submission.
- When you loose connection during the right and left ear sound tests the app will load indefinitely after clicking the start test button.
- If you are offline and click the send button after a test, the send and main menu buttons switches colors.

**Comment by dansvend** [comment]
Thanks for revieweing and testing🙏

I have not done anything about the feedback, I forgot it, but I will implement it there as well😊

Did you test on the simulator or on the app in TestFlight?

Testing the network functionality does not work properly on the simulator so the build in TestFlight that belongs to the currently latest commit om this branch is version '0.2.14' with build numer '20221213.1'.

**Comment by nesadrian** [comment]
Alright. I only tested in simulator so that explains those issues. Give me a heads up when you commit and i'll review :)

**Comment by dansvend** [comment]
@nesadrian 

The link to the `FeedbackScreen` is now hidden from the `SettingsScreen` when the user is offline. This can be tested in version `0.2.14` with build number `20221213.2`.

File: screens/TestScreen.tsx [hunk 1]
```diff
@@ -93,9 +87,16 @@ class TestScreen extends Component {
...
+    isConnected: null,
+    netInfoEventListener: null,
   };
 
   componentDidMount() {
+    this.setState({
+      netInfoEventListener: NetInfo.addEventListener(({ isConnected }) =>
+        this.setState({ isConnected })
+      ),
+    });
```
Comment (nesadrian) [inline screens/TestScreen.tsx:99]: Move out of setState

**Pull request merged**